src/ExpressionParser/Infix/ArgumentsTrait.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\ExpressionParser\Infix;
+
+use Twig\Error\SyntaxError;
+use Twig\Node\Expression\ArrayExpression;
+use Twig\Node\Expression\Unary\SpreadUnary;
+use Twig\Node\Expression\Variable\ContextVariable;
+use Twig\Node\Expression\Variable\LocalVariable;
+use Twig\Node\Nodes;
+use Twig\Parser;
+use Twig\Token;
+
+trait ArgumentsTrait
+{
+    protected function parseCallableArguments(Parser $parser, int $line, bool $parseOpenParenthesis = true): ArrayExpression
```

Comment:
should those be protected or private ? I find it weird that using the trait automatically forces the class to have some protected API (a class *wanting* to have it protected or public can still override the visibility when using the trait)